livestyled/client.py

```diff
@@ -428,3 +430,72 @@ def create_news(
             payload
         )
         return NewsSchema().load(new_team)
+
+    def get_merchant_account(
+            self,
+            id: int or str,
+    ) -> Generator[Dict, None, None]:
+        return self._get_resource(
+            id,
+            MerchantAccountSchema,
+        )
+
+    def get_merchant_accounts(
+            self,
+            status: str or None = None,
+            payment_gateway: str or None = None,
```

Comment:
@Edvinas9 maybe better use one parameter `filter` ? what do you think ?

Reply:
@Edvinas9  we should make use of filters so tomorrow if we need to search by label, we won't need to make a PR just for that.